buildlib/pyverbs_functions.cmake

```diff
@@ -8,7 +8,7 @@ function(build_module_from_cfiles PY_MODULE MODULE_NAME ALL_CFILES LINKER_FLAGS)
   string(REGEX REPLACE "\\.so$" "" SONAME "${MODULE_NAME}${CMAKE_PYTHON_SO_SUFFIX}")
   add_library(${SONAME} SHARED ${ALL_CFILES})
   set_target_properties(${SONAME} PROPERTIES
-    COMPILE_FLAGS "${CMAKE_C_FLAGS} -fPIC -fno-strict-aliasing -Wno-unused-function -Wno-redundant-decls -Wno-shadow -Wno-cast-function-type -Wno-implicit-fallthrough -Wno-unknown-warning -Wno-unknown-warning-option -Wno-deprecated-declarations ${NO_VAR_TRACKING_FLAGS}"
+    COMPILE_FLAGS "${CMAKE_C_FLAGS} -fPIC -fno-strict-aliasing -Wno-unused-function -Wno-redundant-decls -Wno-shadow -Wno-cast-function-type -Wno-implicit-fallthrough -Wno-deprecated-declarations ${NO_VAR_TRACKING_FLAGS}"
```

Comment:
This is probably going to break something else.. I guess you need to test the flags for compatability.

Reply:
I don't see the mentioned c11 warnings/notes now.
Anyway, I understand that it's not related to FC39 specifically, so I'm removing this commit from this PR.